srcpkgs/muon/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'muon'
+pkgname=muon
+version=0.1.0
+revision=1
+build_style=meson
+build_helper=qemu
+configure_args="
+ -Dlibcurl=enabled
+ -Dlibarchive=enabled
+ -Dlibpkgconf=enabled
+ -Dsamurai=enabled
```

Comment:
> embed samurai into the muon executable

This is most likely not what we want.  With this set to enabled you now have 2 copies of samurai, one embedded into muon and one pulled in as runtime dependency.

Reply:
It statically links against libsamurai, which is not part of the official release, but is provided by a [fork](https://github.com/annacrombie/samurai/blob/libsamu/meson.build#L30).

Reply:
But you're right, I'll remove this flag.

Reply:
The main purpose of libsamu is so that muon can serve as a single-file statically linked redistributable which does everything you need. It's doubtful that that is generally valuable here, so I'd agree with removing that one option.